docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties 
+
+[role="_abstract"]
+Use this page to work with yaml files of devfile properties. See the following table of contents to go to the devfile property you want to work with:
```

Comment:
- Two times "work with"
- "yaml files that have devfile properties" maybe?
- The first sentence is very vague. How can the user use this page to work with yaml files? Is this page explaining how to edit them in various ways? This comes down to "what does 'work with' mean"? Which will also help in finding a replacement fro this expression. I think a proper description of what is actually going on here would help.

Reply:
Okay cool, let me flush out the abstract with your suggestions there, then see what you think about the revision!